inst/templates/layout.html

```diff
@@ -3,9 +3,12 @@
 <html lang="en">
   <head>
   {{{ head }}}
+  {{{ in-header }}}
```

Comment:
Maybe we should call this `after-head`? It would be weird not to have "before-head" but the order of the head elements isn't important like body elements.

Reply:
I thought it'd be less confusing for bookdown users to use the same names. :thinking:

Reply:
Oooooh, what if we support both?

Reply:
i.e. just add `{{{after-head}}}` as well

Reply:
Wouldn't this make the docs more complicated?

Reply:
We don't mention that you can use both, we just say use `after-head`/`in-header` to insert additional head statements after the defaults generated by pkgdown.